jekyll/_cci2/using-macos.md

```diff
@@ -263,19 +248,19 @@ React Native projects can be built on CircleCI using `macos` and `docker` execut
 ### Creating a `config.yml` File
 {: #creating-a-configyml-file }
 
-The most flexible way to customize your build is to modify the CircleCI configuration for your project in `.circleci/config.yml`. This allows you to run arbitrary bash commands as well as use built-in features such as workspaces and caching. See the [Configuring CircleCI]( {{ site.baseurl }}/configuration-reference/) documentation for a detailed description of the structure of the `.circleci/config.yml` file.
+The most flexible way to customize your build is to modify the CircleCI configuration for your project in `.circleci/config.yml`. This allows you to run arbitrary bash commands as well as use built-in features such as [workspaces]({{site.baseurl}}/workspaces) and [caching]({{site.baseurl}}/caching). See the [Configuring CircleCI]({{site.baseurl}}/configuration-reference/) documentation for a detailed description of the structure of the `.circleci/config.yml` file.
 
 ## Using Multiple Executor Types (macOS + Docker)
```

Comment:
@devinmarieb I think what you have done with this example is totally fine. IMO we could just remove this whole section though and we can add more generic information on using multiple executors in a single pipeline or workflow to the Intro to Execution Envs doc. What do you think? We have this example we could use: /docs/sample-config#sample-configuration-with-multiple-executor-types ... There are some other things we've talked about doing in that intro doc so I'm happy to make a ticket

Reply:
@rosieyohannan Yeah that could be a solution. The second example uses community driven Docker images (from Docker community not ours) and while they have hundreds of thousands of downloads, they haven't been updated in years (one of them over four years). Another convo around using images/orbs in examples that are community driven and not officially maintained I guess.